zone/gm_commands/summon.cpp

```diff
@@ -92,6 +89,12 @@ void command_summon(Client *c, const Seperator *sep)
 			2,
 			GMSummon
 		);
+	} else {
```

Comment:
Can we simply return inside `if (target->IsClient()) {` and not rely on the `else` block here?

If `if (target->IsClient()) {` is true we should never reach this code which would have the same effect.

Materially this doesn't change the end result just is a bit cleaner to read

```cpp
// example
if (target->IsClient()) {
	// do stuff
	return
}

// old else code
target->GMMove(c->GetPosition());
if (target->IsNPC()) {
	target->CastToNPC()->SaveGuardSpot(glm::vec4(0.0f));
}
```

Reply:
Yeah no problem, I do that everywhere else, just didn't here.